src/ui/zcl_abapgit_gui_page_code_insp.clas.abap

```diff
@@ -231,9 +241,22 @@ CLASS zcl_abapgit_gui_page_code_insp IMPLEMENTATION.
           " we need to refresh as the source might have changed
           lo_repo_online->refresh( ).
 
+          READ TABLE mt_result TRANSPORTING NO FIELDS WITH KEY kind = 'E'.
```

Comment:
my suggestion is to keep the whole mt_result, then you can for example show the number of errors in the hover text

Reply:
as in export it to the page insstead of calculating the status here

Reply:
Number of errors: yeah, good idea, will check
Calculating status ... actually I think it belongs to SCI class more then to stage ... what I wanted to do initially is to separate sci runner. Because now a service (controller) asks the sci page (view) about the result of sci (model) ... (?!?!?!) But I think it is a subject for another PR